test/e2e/snaps/test-snap-cronjob-duration.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('Test Snap Cronjob Duration', function () {
         await headerNavbar.checkNotificationCountInMenuOption(1);
 
         // This click will close the menu.
-        await headerNavbar.openThreeDotMenu();
+        await headerNavbar.mouseClickOnThreeDotMenu();
```

Comment:
It looks like `openThreeDotMenu` accounts for differences on Firefox vs. Chrome. Do we not need to do that anymore?

Reply:
The original test had flaw itself, it called both only 1 click really mattered because of the popover bug